app/views/admin/statistics_announcements/show/_main.html.erb

```diff
@@ -0,0 +1,143 @@
+<div class="app-view-summary__main">
+  <section class="app-view-summary__section">
+    <p class="govuk-body-lead">
+      <%= @statistics_announcement.summary %>
+    </p>
+
+    <%=
+      render "govuk_publishing_components/components/summary_list", {
```

Comment:
<%=  render "govuk_publishing_components/components/summary_list", {

This can be on the same line